pos_partner_ref/static/src/js/ClientDetailsEdit.esm.js

```diff
@@ -0,0 +1,13 @@
+/** @odoo-module **/
+import {PartnerDetailsEdit} from "@point_of_sale/app/screens/partner_list/partner_editor/partner_editor";
+import {patch} from "@web/core/utils/patch";
+
+patch(PartnerDetailsEdit.prototype, {
+    setup() {
+        super.setup(...arguments);
+        this.changes.ref = this.props.partner.ref || null;
+    },
+    _today() {
+        return new Date().toISOString().split("T")[0];
```

Comment:
Is this needed at all or a remanent of copied code :sweat_smile: ?

Reply:
thanks